packages/dashboard/src/components/__tests__/FeatureSidebar.test.tsx

```
@@ -0,0 +1,28 @@
+/// <reference types="@testing-library/jest-dom" />
+import { describe, it, expect } from 'vitest';
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { FeatureSidebar, type FeatureSidebarListItem } from '../FeatureSidebar';
```

ESLint

[error] '../FeatureSidebar' import is restricted from being used by a pattern. Use relative imports only within the same folder. Use dashboard #... aliases for imports outside the current folder.

(no-restricted-imports)